core/server/services/mega/mega.js

```diff
@@ -187,7 +202,7 @@ async function listener(emailModel, options) {
     });
 
     let meta = [];
-    let error;
+    let error = null;
```

Comment:
Was initially inclined to set this value in `retryFailedEmail` method, but here it seems more suitable as we already modify `error` property anyways.